### PR TITLE
fix(chatbar): fix emoji sending issue AP-474

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

fixed for not inputting emojis in chatbar.

**Which issue(s) this PR fixes** 🔨

AP-474

### CODE REVIEW

File: components/views/chat/chatbar/Chatbar.vue [hunk 1]
```diff
@@ -240,13 +240,12 @@ export default Vue.extend({
...
       const messageBox = this.$refs.messageuser as HTMLElement
-      if (collapseToEnd) {
-        let sel = window.getSelection()
-        sel?.selectAllChildren(messageBox)
-        sel?.collapseToEnd()
-      }
+      messageBox.innerHTML = this.value
+      let sel = window.getSelection()
+      sel?.selectAllChildren(messageBox)
+      sel?.collapseToEnd()
```
Comment (WanderingHogan) [inline components/views/chat/chatbar/Chatbar.vue:248]: I think this is forcing the cursor to the end of the input any time you move the cursor back in the text and input anything

File: components/views/chat/chatbar/Chatbar.vue [hunk 2]
```diff
@@ -195,8 +195,8 @@ export default Vue.extend({
         messageBox.innerText &&
         messageBox.innerText.length > this.$Config.chat.maxChars + 1
       ) {
-        /* remove updateText() here because when this.value is changed it is automatically called */
         messageBox.innerText = messageBox.innerText.slice(0, -1)
+        this.updateText()
```
Comment (WanderingHogan) [inline components/views/chat/chatbar/Chatbar.vue:199]: do we need this and the watch below?

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
A question and an issue int he comments

File: components/views/chat/chatbar/Chatbar.vue (hunk 2 above)
Comment (davidchan1219) [reply components/views/chat/chatbar/Chatbar.vue:199]: yes

File: components/views/chat/chatbar/Chatbar.vue (hunk 1 above)
Comment (davidchan1219) [reply components/views/chat/chatbar/Chatbar.vue:248]: yes, this code was written by the old developer.

Comment (WanderingHogan) [reply components/views/chat/chatbar/Chatbar.vue:248]: I get that, but we are all growing and evolving this thing together. In this particular case, removing if(collapseEnd)  means that every character added will force the cursor to the end and we can't accept this pull request that fixes one bug and introduces another

Comment (davidchan1219) [reply components/views/chat/chatbar/Chatbar.vue:248]: got it, let me find the other solution

**Review by phillsatellite — APPROVED** [review]

**Comment by stavares843** [comment]
@davidchan1219 thanks for taking a look! @molimauro is gonna fix this on a PR is working on 🔨 

so we are closing this one

**Pull request closed without merging**